sql/moz-fx-data-shared-prod/firefox_desktop_derived/newtab_clients_daily_aggregates_v2/schema.yaml

```diff
@@ -0,0 +1,157 @@
+fields:
+- name: submission_date
+  type: DATE
+  mode: NULLABLE
+  description: Date of aggregation
+- name: app_version
+  type: INTEGER
+  mode: NULLABLE
+  description: Fx major version number
+- name: os
+  type: STRING
+  mode: NULLABLE
+- name: channel
+  type: STRING
+  mode: NULLABLE
+- name: locale
+  type: STRING
+  mode: NULLABLE
+- name: country
+  type: STRING
+  mode: NULLABLE
+- name: homepage_category
+  type: STRING
+  mode: NULLABLE
+  description: Setting for home page.
```

Comment:
this description and the next one could explain how are homepage and newtab categories different

Reply:
@lucia-vargas-a Good suggestion. I will try to find out